pyccl/halos/profiles/einasto.py

```diff
@@ -37,16 +38,27 @@ class HaloProfileEinasto(HaloProfileMatter):
             concentration-mass relation to use with this profile.
         truncated (:obj:`bool`): set to ``True`` if the profile should be
             truncated at :math:`r = r_\\Delta`.
+        projected_quad (:obj:`bool`): set to ``True`` to calculate the
+            projected profile with numerical integration.
         alpha (:obj:`float` or :obj:`str`): :math:`\\alpha` parameter, or
             set to ``'cosmo'`` to calculate the value from cosmology.
     """
     __repr_attrs__ = __eq_attrs__ = (
         "truncated", "alpha", "mass_def", "concentration", "precision_fftlog",)
```

Comment:
can you add `"projected_quad"` to the list of attributes here?

Reply:
Fixed.